### PR TITLE
[Doc] Fix typos in docs

### PR DESCRIPTION
**Pull request by DarkLight1337**
I missed these in #10621 and #10470

### CODE REVIEW

File: docs/source/serving/compatibility_matrix.rst [hunk 1]
```diff
@@ -393,7 +393,7 @@ Feature x Hardware
      - ✅
      - ✅
      - ✅
-     - ✗
+     - ?
```
Comment (russellb) [inline docs/source/serving/compatibility_matrix.rst:396]: did you intend to include this?  just checking since it doesn't look related to the PR title and wasn't mentioned in the summary

Comment (DarkLight1337) [reply docs/source/serving/compatibility_matrix.rst:396]: Yes, let me update the title.

**Review by youkaichao — APPROVED** [review]
